vortex/vortex-tools.desktop

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Name=Update Vortex Game Library
```

Comment:
**question:** Would `Setup Vortex Games` or `Setup Vortex Game Libraries` be more clear here?

Seems to be doing more than just 'updating'.

Reply:
Oops, thats my bad I didn't actually change that at all haha